test/utils-test.js

```diff
@@ -179,4 +179,40 @@ describe('utils tests', function(){
             should(res).eql(false);
         });
     });
+    describe('When calling getLogLevel', () => {
+        it('Should return info if levels is undefined', () => {
+            let result = utils.getLogLevel(200, undefined);
+            should(result).eql('info');
+        });
+        it('Should return info if levels is empty array', () => {
+            let result = utils.getLogLevel(200, []);
+            should(result).eql('info');
+        });
+        it('Should return info if exact match but levelMap value is not valid level', () => {
+            let result = utils.getLogLevel(200, { '200': 'not-valid' });
+            should(result).eql('info');
+        })
+        it('Should return correct exact match of status with level map', () => {
+            let result = utils.getLogLevel(200, { '200': 'error' });
+            should(result).eql('error');
+        });
+        it('Should fallback to status group if exact match not found', () => {
```

Comment:
I would add one more test for an exact match while group is also configured

Reply:
done